config/locales/views.de.yml

```
@@ -405,6 +412,21 @@ de:
       attachment:
         subject: 'Nachrichten Betreff:'
 
+  second_factor_authentication:
+    flash:
+      success:
+        registered: Zwei Faktor Authentifizierung erfolgreich aktiviert
+        signed_in: Erfolgreich eingeloggt
+
+      failure: One-Time Code inkorrekt
+    new:
+      title: Zwei Faktor Authentifizierung
+      hint_registered_html: "Geben Sie bitte den angezeigten Code auf Ihrer %{link} an um Ihre Identität zu bestätigen."
+      hint_unregistered_html: "Scannen Sie bitte den QR Code mit ihrem Smartphone mittels der %{link} und geben Sie den angezeigten Code an um die zwei Faktor Authentifizierung zu aktivieren."
```

```suggestion
      hint_unregistered_html: "Scannen Sie bitte den QR Code mit ihrem Smartphone mittels der %{link} und geben Sie den angezeigten Code an um die Zwei-Faktor-Authentifizierung zu aktivieren."
```
https://de.wikipedia.org/wiki/Zwei-Faktor-Authentisierung